presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHiveLocal.java

```diff
@@ -98,30 +99,35 @@ protected ConnectorTableHandle getTableHandle(ConnectorMetadata metadata, Schema
     }
 
     @Override
+    @Test(enabled = false)
     public void testGetAllTableNames()
     {
         throw new SkipException("Test disabled for this subclass");
     }
 
     @Override
+    @Test(enabled = false)
     public void testGetAllTableColumns()
     {
         throw new SkipException("Test disabled for this subclass");
     }
 
     @Override
+    @Test(enabled = false)
     public void testGetAllTableColumnsInSchema()
     {
         throw new SkipException("Test disabled for this subclass");
```

Comment:
Maybe we should disallow using `SkipException` then.

Reply:
SkipException isn't bad.. has some pros and cons...
but we sometimes (oftentimes?) override with an empty body, which is worst of all